### ISSUE TITLE
Improve loading screens

### ISSUE THREAD

**Issue by matthiaskern** [Sketch-sh/sketch-sh#133]
Loading screens should resemble the actual UI it is "morphing" to better.

**Comment by thangngoc89** [comment]
Are we talking about the loading screens creating with ReactContentLoader?

**Comment by matthiaskern** [comment]
Thinking about these: 
![image](https://user-images.githubusercontent.com/9108002/44509275-f0911280-a6b9-11e8-8fe2-4e52aa069a52.png)
Could be adjusted to look more like the screen it is loading,e.g.:
![image](https://user-images.githubusercontent.com/9108002/44509306-0dc5e100-a6ba-11e8-95b6-67ae8c4bdf89.png)

**Comment by thangngoc89** [comment]
yep. that's good ideas. Morphing only works when it looks exactly like the final screen

**Comment by nimish-gupta** [comment]
@thangngoc89 can i go forward with this issue?

**Comment by thangngoc89** [comment]
@matthiaskern yeah sure. take a look at this thread  https://twitter.com/_chenglou/status/1034530454597062656 :P 

I found this library with css only approach https://github.com/zalog/placeholder-loading . It looks very promising and we can put the block inside sketch's css to have the exact sizes/spacing

**Comment by nimish-gupta** [comment]
@thangngoc89 can you tell me on which of the screen you want loading screen? And provide a little bit more context for the issue?

**Comment by thangngoc89** [comment]
@nimish-gupta there is currently 2 loading screens. 1 is the sketch viewer with editor. 

https://github.com/Sketch-sh/sketch-sh/blob/9a0b0011d06da4f33b171b806126e13a780ad54b/client/src_editor/Editor_NotePlaceholder.re

And the other one is SketchList (homepage when logined)
https://github.com/Sketch-sh/sketch-sh/blob/4c7fafec4784a4b8a9dda290f9ad3a0df2561857/client/src/components/UI_SketchList.re#L94

I would like to improve them first because currently they are not correct with the final shape (and it's worst then no loading screen at all)

**Comment by nimish-gupta** [comment]
ok got it
